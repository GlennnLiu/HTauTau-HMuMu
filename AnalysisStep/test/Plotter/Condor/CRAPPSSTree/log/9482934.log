000 (9482934.007.000) 07/27 15:05:28 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_8624_ecd7>
...
040 (9482934.007.000) 07/27 15:06:57 Started transferring input files
	Transferring to host: <188.184.163.193:9618?addrs=188.184.163.193-9618+[2001-1458-301-4d--100-bb]-9618&alias=b7s12p3439.cern.ch&noUDP&sock=slot1_12_244344_33c9_57720>
...
040 (9482934.007.000) 07/27 15:06:57 Finished transferring input files
...
001 (9482934.007.000) 07/27 15:06:58 Job executing on host: <188.184.163.193:9618?addrs=188.184.163.193-9618+[2001-1458-301-4d--100-bb]-9618&alias=b7s12p3439.cern.ch&noUDP&sock=startd_143646_c959>
...
006 (9482934.007.000) 07/27 15:07:07 Image size of job updated: 172356
	88  -  MemoryUsage of job (MB)
	89780  -  ResidentSetSize of job (KB)
...
040 (9482934.007.000) 07/27 15:07:08 Started transferring output files
...
040 (9482934.007.000) 07/27 15:07:08 Finished transferring output files
...
005 (9482934.007.000) 07/27 15:07:08 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:06, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:06, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	23923  -  Run Bytes Sent By Job
	346  -  Run Bytes Received By Job
	23923  -  Total Bytes Sent By Job
	346  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       37        1   2934729 
	   Memory (MB)          :       88     2000      2000 

	Job terminated of its own accord at 2023-07-27T13:07:08Z.
...
