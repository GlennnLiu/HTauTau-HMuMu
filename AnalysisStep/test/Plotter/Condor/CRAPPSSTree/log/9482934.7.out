[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.diBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.triBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.top.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.EWKZee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.EWKZtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.EWKZmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.DYee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.DYtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.DYmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.Wjet.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.TTfake.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.Data.root
[INFO] Total MC summed
[INFO] Saved Histos to /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPSSTree.TotalMC.root
